hw/drivers/uart/max3107/max3107_0/pkg.yml

```diff
@@ -0,0 +1,27 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+pkg.name: hw/drivers/uart/max3107/max3107_0
+pkg.description: Package adds single instance of MAX3107 device.
+pkg.author: "Apache Mynewt"
+pkg.keywords:
+    - uart
+
+pkg.init:
+    max3107_0_init: 'MYNEWT_VAL(MAX3107_0_SYSINIT_STAGE)'
```

Comment:
this function is not going to be any different for different instances. So, I would just name it `max3107_init` and also the syscfg should be named `MAX3107_SYSINIT_STAGE`

Reply:
I disagree